---
layout: member
category: student
title: Bonface Onyango
image: bonface.png
role: PhD Student
permalink: 'team/bonface'
social:
    twitter: https://twitter.com/
    linkedin: https://www.linkedin.com/in/bonface-onyango-927145161/
    google-scholar: https://scholar.google.ca/citations?user=iXFHiXYAAAAJ&hl=en
    github: https://github.com/bonfaceonyango
    website:
    orcid: https://orcid.org/0000-0002-9360-3788
    research-gate: https://www.researchgate.net/
education:
 - Interdisciplinary PhD student (Started Sep. 2024)
 - MSc Bioinformatics, Pwani University, Kenya 
 - B.Sc. Biochemistry, Egerton University, Kenya
interests:
- Comparative genomics
- Evolutionary Biology
- Machine learning
- Antimicrobial resistance
- Bioinformatics software development
---
Antimicrobial resistance (AMR) genes frequently spread via horizontal gene transfer, alongside neighboring genes. However, most computational methods for antimicrobial resistance gene detection mainly rely on comparing a suspected AMR gene to predefined AMR genes in reference databases, without considering the neighboring genes. Morever, the databases are often built from well-studied or clinically relevant organisms, which limits their ability to generalize—particularly in identifying novel or divergent AMR genes in distantly related and understudied organisms. Therefore, my research focuses on developing comparative genomics and machine learning approaches that integrate neighboring gene order information. The goal is to improve the detection of hidden AMR threats—so-called “unknown unknowns”—and to better understand the evolution of AMR gene neighborhoods, potentially reducing the growing burden of AMR.